src/components/button/button.js

```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components';
+
+export default function Button({disabled, buildId, buildTime, onHandleClick, buttonText}) {
+  return (
+    <StyledButton disabled={disabled} data-buildid={buildId} data-buildtime={buildTime} onClick={onHandleClick}>
```

Comment:
The `data-buildId` and `data-buildtime` are not button attributes and would be ignored by the styled-component. I think you can just remove them 🤔 
```suggestion
    <StyledButton disabled={disabled}  onClick={onHandleClick}>
```

Reply:
i put type="button" to it